### PR TITLE
Resolved Login Portal Toggle #2977

### LINKED ISSUE #2977: Issue in Admin and User Login buttons when in focus at Login Page
**Describe the bug**
The Admin and User buttons in the Login page have a font color problem(same font as background) when in focus. This causes the makes the button contents invisible.

**To Reproduce**

1. Go to Login Page
2. Click the User/Admin button 

**Expected behavior**
The font color of the button in focus should not be same as the background colour of the button.

**Actual behavior**
The font colour of the button that is clicked is changed to that of its background.

**Screenshots**

https://github.com/user-attachments/assets/80959189-6089-47cc-be16-1d65a63575eb

### PR DESCRIPTION
**Pull request by aadhil2k4**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #2977 

**Did you add tests for your changes?**

N/A, only CSS changes

**Snapshots/Videos:**

https://github.com/user-attachments/assets/4a90374d-0dd2-4cc8-be37-9312a6b41795

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **Style**
	- Updated color and background color for active links and their hover states.
	- Introduced a new CSS variable for consistent theming of toggle buttons.

### CODE REVIEW

File: src/components/LoginPortalToggle/LoginPortalToggle.module.css [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
-  background-color: var(--active-button-bg);
+  background-color: #31bb6b;
 }
 
 .activeLink:hover {
-  color: var(--bs-white);
-  background-color: var(--active-hover);
+  color: var(--bs-btn-bg);
+  background-color: #31bb6b;
   border: 1px solid var(--bs-primary);
```
[the remark this answers is not in the document]
Comment (aadhil2k4) [reply src/components/LoginPortalToggle/LoginPortalToggle.module.css]: making these suggested changes results to the same issue for which the PR was created

**Comment by palisadoes** [comment]
Please make sure Code rabbit approves your changes

File: src/components/LoginPortalToggle/LoginPortalToggle.module.css [hunk 2]
```diff
@@ -22,13 +22,13 @@
 }
 
 .activeLink {
-  color: var(--bs-white);
+  color: white;
   border: 1px solid var(--bs-primary);
-  background-color: var(--active-button-bg);
+  background-color: #31bb6b;
```
Comment (palisadoes) [inline src/components/LoginPortalToggle/LoginPortalToggle.module.css:27]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

File: src/components/LoginPortalToggle/LoginPortalToggle.module.css [hunk 3]
```diff
@@ -22,13 +22,13 @@
...
 }
 
 .activeLink:hover {
-  color: var(--bs-white);
-  background-color: var(--active-hover);
+  color: white;
+  background-color: #31bb6b;
```
Comment (palisadoes) [inline src/components/LoginPortalToggle/LoginPortalToggle.module.css:32]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

File: src/components/LoginPortalToggle/LoginPortalToggle.module.css [hunk 4]
```diff
@@ -22,13 +22,13 @@
 }
 
 .activeLink {
-  color: var(--bs-white);
+  color: white;
   border: 1px solid var(--bs-primary);
-  background-color: var(--active-button-bg);
+  background-color: #1e4e8c;
```
Comment (palisadoes) [inline src/components/LoginPortalToggle/LoginPortalToggle.module.css:27]: Convert to a variable. There should be some preexisting ones, if not please create it.

File: src/components/LoginPortalToggle/LoginPortalToggle.module.css [hunk 5]
```diff
@@ -22,13 +22,13 @@
...
 }
 
 .activeLink:hover {
   color: var(--bs-white);
-  background-color: var(--active-hover);
+  background-color: #1e4e8c;
```
Comment (palisadoes) [inline src/components/LoginPortalToggle/LoginPortalToggle.module.css:32]: Convert to a variable. There should be some preexisting ones, if not please create it.

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

**Review by palisadoes — APPROVED** [review]

**Pull request merged**